src/Ast.ml

```diff
@@ -14,15 +14,19 @@
 open Migrate_ast
 open Parsetree
 
-let init, register_reset, leading_nested_match_parens =
+let init, register_reset, leading_nested_match_parens, parens_ite =
   let l = ref [] in
-  let leading_nested_match_parens = ref false in
+  let leading_nested_match_parens =
+    ref Conf.(default_profile.leading_nested_match_parens)
+  in
+  let parens_ite = ref Conf.(default_profile.parens_ite) in
```

Comment:
Why do we rely on the default_profile here ? I would have expected the init value of the reference to be meaningless

Reply:
You are right, I will revert this